boards/msb-430h/include/board.h

```diff
@@ -18,8 +18,8 @@
  * @author      Freie Universität Berlin, Computer Systems & Telematics, FeuerWhere project
  */
 
-#ifndef _MSB_BOARD_H
-#define _MSB_BOARD_H
+#ifndef MSB_BOARD_H
+#define MSB_BOARD_H
```

Comment:
All I did was `git grep -l 'BOARD_H' | xargs sed -i 's/BOARD_H/BOARD_H/' and the same for `PERIPH_CONF_H`. But yes, I can change the rest, too.